src/orange/orangeinp/ConvexRegion.cc

```diff
@@ -487,6 +487,22 @@ void GenTrap::build(ConvexSurfaceBuilder& insert_surface) const
                            GeneralQuadric{abc, def, ghi, offset});
         }
     }
+
+    // Construct exterior bounding box
+    BBox exterior_bbox;
+    for (VecReal2 const* p : {&lo_, &hi_})
+    {
+        for (Real2 const& xy : *p)
+        {
+            for (auto ax : {Axis::x, Axis::y})
+            {
+                exterior_bbox.grow(ax, xy[to_int(ax)]);
+            }
+        }
+    }
+    exterior_bbox.grow(Axis::z, -hz_);
+    exterior_bbox.grow(Axis::z, hz_);
```

Comment:
IIUC, it is okay to save some cycles this way:
```suggestion
    exterior_bbox.grow(Bound::lo, Axis::z, -hz_);
    exterior_bbox.grow(Bound::hi, Axis::z, hz_);
```